apps/web/utils/llms/index.ts

```diff
@@ -65,6 +66,17 @@ function getModel({ aiProvider, aiModel, aiApiKey }: UserAIFields) {
     };
   }
 
+  if (provider === Provider.OLLAMA) {
+    const model = aiModel || Model.OLLAMA_MODEL;
+    return {
+      provider: Provider.OLLAMA,
+      model,
+      llmModel: createOllama({
+        baseURL: "https://api.ollama.com",
```

Comment:
Would be better to set this as an env var so that people that self-host could even point this to their local Ollama if they want.

Reply:
Also what even is https://api.ollama.com/?

Reply:
Yes, that's right! It’s from the Ollama documentation. I figured only tech-savvy users would be using Ollama, so they’d know that this would be replaced.

Regarding the comment above, claude also has multiple models. how should we handle this adn the API url? should we mention it in the README?

Reply:
Great way to confuse tech savy users 😂
Env vars are used for fields that need to be replaced